react/tmpl/src/index.js

```diff
@@ -8,6 +8,9 @@ let mainWindow;
 
 const isDevMode = process.execPath.match(/[\\/]electron/);
 
+let electronReload = require('electron-reload');
+electronReload(__dirname);
+
```

Comment:
Just starting to learning electron/node.js, so I like this tip (use of `electron-reload`), already using it, because I was missing some live-reload here 👍 .
But... this `let` (and the rest you add in this commit) should be a `const`, right? Because it doesn't changes it's value over time... And, you know, also ESLint warns about it here. And same for the space between the require and other not `require` lines, in this and the rest of the modified templates. Am I right?